docs/content/en/docs/getting-started/install/_index.md

```diff
@@ -5,36 +5,38 @@ description: >
   Steps for setting up the Admin Machine
 ---
 
+{{% alert title="Warning" color="warning" %}}
+The Administrative machine should not be ephemeral. Artifacts created during cluster operations will be stored on the Administrative machine. Losing this machine could result in loss of cluster information.
+{{% /alert %}}
+
 EKS Anywhere will create and manage Kubernetes clusters on multiple providers.
 Currently we support creating development clusters locally using Docker and production clusters from providers listed on the [providers]({{< relref "/docs/getting-started/chooseprovider/" >}}) page.
 
-Creating an EKS Anywhere cluster begins with setting up an Administrative machine where you will run Docker and add some binaries.
-From there, you create the cluster for your chosen provider.
+Creating an EKS Anywhere cluster begins with setting up an Administrative machine where you will run all EKS Anywhere lifecycle operations as well as Docker, `kubectl` and the prerequisite utilites.
```

Comment:
Nit: change to present tense. "Creating an EKS Anywhere cluster beings with setting up an Admin machine where you run EKS Anywhere cluster lifecycle operations ..."

Reply:
Updated tense.